File: ios/Classes/UnityMediationConstants.swift
struct UnityMediationConstants {
    static let MAIN_CHANNEL = "com.rebeloid.unity_mediation";
    static let VIDEO_AD_CHANNEL = MAIN_CHANNEL + "/videoAd";
    
    static let AD_UNIT_ID_PARAMETER = "adUnitId";
    static let ERROR_CODE_PARAMETER = "errorCode";
    static let ERROR_MESSAGE_PARAMETER = "errorMessage";
    
    //initialize
    static let INIT_METHOD = "init";
    static let GAME_ID_PARAMETER = "gameId";
    static let INIT_COMPLETE_METHOD = "initComplete";
    static let INIT_FAILED_METHOD = "initFailed";
    
    //load
    static let LOAD_REWARDED_AD_METHOD = "loadRewardedAd";
    static let LOAD_INTERSTITIAL_AD_METHOD = "loadInterstitialAd";
    static let LOAD_COMPLETE_METHOD = "loadComplete";
    static let LOAD_FAILED_METHOD = "loadFailed";
    
    //show
    static let SHOW_REWARDED_AD_METHOD = "showRewardedAd";
    static let SHOW_INTERSTITIAL_AD_METHOD = "showInterstitialAd";
    static let SHOW_REWARDED_METHOD = "showRewarded";
    static let SHOW_FAILED_METHOD = "showFailed";
    static let SHOW_START_METHOD = "showStart";
    static let SHOW_CLOSED_METHOD = "showClosed";
    static let SHOW_CLICK_METHOD = "showClick";
    static let  REWARD_TYPE_PARAMETER = "rewardType";
    static let  REWARD_AMOUNT_PARAMETER = "rewardAmountType";
    
}

File: android/src/main/java/com/rebeloid/unity_mediation/UnityMediationConstants.java
package com.rebeloid.unity_mediation;

public final class UnityMediationConstants {

    public static final String MAIN_CHANNEL = "com.rebeloid.unity_mediation";
    public static final String VIDEO_AD_CHANNEL = MAIN_CHANNEL + "/videoAd";
    public static final String AD_UNIT_ID_PARAMETER = "adUnitId";
    public static final String ERROR_CODE_PARAMETER = "errorCode";
    public static final String ERROR_MESSAGE_PARAMETER = "errorMessage";

    //initialize
    public static final String INIT_METHOD = "init";
    public static final String INIT_STATE_METHOD = "initState";
    public static final String GAME_ID_PARAMETER = "gameId";
    public static final String INIT_COMPLETE_METHOD = "initComplete";
    public static final String INIT_FAILED_METHOD = "initFailed";

    //load
    public static final String LOAD_REWARDED_AD_METHOD = "loadRewardedAd";
    public static final String LOAD_INTERSTITIAL_AD_METHOD = "loadInterstitialAd";
    public static final String LOAD_COMPLETE_METHOD = "loadComplete";
    public static final String LOAD_FAILED_METHOD = "loadFailed";

    //show
    public static final String SHOW_REWARDED_AD_METHOD = "showRewardedAd";
    public static final String SHOW_INTERSTITIAL_AD_METHOD = "showInterstitialAd";
    public static final String SHOW_REWARDED_METHOD = "showRewarded";
    public static final String SHOW_FAILED_METHOD = "showFailed";
    public static final String SHOW_START_METHOD = "showStart";
    public static final String SHOW_CLOSED_METHOD = "showClosed";
    public static final String SHOW_CLICK_METHOD = "showClick";
    public static final String REWARD_TYPE_PARAMETER = "rewardType";
    public static final String REWARD_AMOUNT_PARAMETER = "rewardAmountType";
    public static final String STS_USER_ID_PARAMETER = "userId";
    public static final String STS_CUSTOMIZED_DATA_PARAMETER = "customizedData";

    //state
    public static final String REWARDED_AD_STATE_METHOD = "rewardedAdState";
    public static final String INTERSTITIAL_AD_STATE_METHOD = "interstitialAdStateMethod";

    private UnityMediationConstants() {
    }
}
